### ISSUE TITLE
H_07 Title in page content correctly tagged is a direct violation of WCAG 1.3.1 Info and relationships.

### ISSUE THREAD

**Issue by wsbeast** [pdf-association/techniques-for-accessible-pdf#11]
H_07 Title in page content correctly tagged is a direct violation of WCAG 1.3.1 Info and relationships. The title is a heading. It is not something else. Definitely not a P tag. 

It is the first heading that all other content relates to. Deciding that role mapping it to a P tag so that it is not distinguishable from any other body content seems beyond logic.

While I understand that assistive technology should correctly voice it as a title, the decision to map it to the most indistinguishable tag in the tags tree (a P tag) defies all logic.

**Comment by wsbeast** [comment]
Scenario....

Title on the Cover is followed by page 2.. Table of Contents. So with your suggestion, the first Heading in this document to someone using assistive technology would be "Table of Contents." This designation would mean that when the user presses H to move to the first heading or Shift H to move back to the First heading depending on where they are, the only heading that would be voiced as the primary heading in the document would be Table of Contents? 

How does this gel with the "Only one H1 in a document" audience? Checkers require the first heading of a document to be an H1. A document cannot start with an H2 and pass PAC. So if we follow this rule, then we cannot call the TOC H2 and then the only H1 in a document would be the TOC? This doesn't make sense.

**Comment by willkil** [comment]
> The title is a heading.

While one can argue this, it's certainly not universally accepted. As evidence, consider the default styles in MS Word, Apple Pages, and LibreOffice. They all have a Title style that is distinct from Heading styles.

> H_07 Title in page content correctly tagged is a direct violation of WCAG 1.3.1 Info and relationships.

1.3.1 says, "Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text." If someone follows H_07, the on-page title "can be programmatically determined" by the using the unmapped `Title` tag, regardless of its role mapping. If it's role mapped to `H1`, then to "programmatically determine" whether or not that `H1` is a title, software would still need to look at the unmapped tag.

> Title on the Cover is followed by page 2... the only heading that would be voiced as the primary heading in the document would be Table of Contents?

I do not follow your scenario. If there is only a title page and Table of Contents and no other content, then I would expect following the technique to yield exactly one heading. If there is other content in the document, then I would expect additional headings after Table of Contents, at least one for each main section listed in that ToC. Those would be `H1`s, like the `H1` for the ToC itself, so hitting H or 1 on the keyboard would traverse them.

> How does this gel with the "Only one H1 in a document" audience?

PDF definitely allows multiple H1s. While using a single H1 may be an accessibility recommendation for HTML, it's not for PDF. While the Accessibility LWG does not recommend restricting documents to a single H1, it's certainly valid for PDF/UA-1 and PDF 1.x. Using that single H1 for the title is even recognized as valid in the related technique [H_F07 Both title and top level headings incorrectly tagged as H1](https://pdfa.org/techniques-for-accessible-pdf/both-title-and-top-level-headings-incorrectly-tagged-as-h1/UA1_Tpdf-H_F07/): "If a title is tagged as H1, however, the main section headings must be tagged as H2."

**Comment by FvdGDTP** [comment]
I see no violation of 1.3.1.
Maybe it is a difference between Europe and the US, but a title here is not considered a heading, if it was it would be part of a TOC including page number. In Europe at least this is not done. A title is a title, it can be meta data and called for when needed. The advice to let it be a P awaiting the PDF2 true title tag is the official advice for government institutions at least in the Netherlands (no matter what Word wants to do). However, you may want to tag it H1, but repeat H1 as you start true header level. But as a title is a different thing than a header (an outline of the document does not contain the document title, at least again not in Europe in general) I see no violation of 1.3.1.

**Comment by macedvisioned** [comment]
[like]  Ed Boon reacted to your message:
________________________________

**Comment by DuffJohnson** [comment]
Closing as resolved.
